### ISSUE TITLE
The cpu usage is different from the task manager.

### ISSUE THREAD

**Issue by kingkong3** [oshi/oshi#2059]
Hello, I'm developing CPU monitoring tools by Numa in Java through the oshi library.

There is a difference between the cpu usage measured by oshi and the cpu usage from Windows Task Manager.

**My server cpu info :**
 
![image](https://user-images.githubusercontent.com/23719336/172278378-80b74a57-a807-420e-8f77-c88598c6cd2f.png)

**My source code :** 

            // Make a map by checking the numa of each processor
            Map<Integer, Integer> cpuInfo = new ConcurrentHashMap<>();
		List<LogicalProcessor> list = cpu.getLogicalProcessors();
		for(LogicalProcessor lp : list) {
			int numa = lp.getNumaNode();
			cpuInfo.put(lp.getProcessorNumber(), numa);
		}

            long[][] oldTicks = cpu.getProcessorCpuLoadTicks();
			
             // sleepInterval : 1000
            Thread.sleep(sleepInterval);
			
            double [] values = cpu.getProcessorCpuLoadBetweenTicks(oldTicks);
			
            Map<Integer, Double> cpuValues = this.getCpuByNuma(cpuInfo, values);

            insertDataBase(cpuValues);

.
.
.

            private Map<Integer, Double> getCpuByNuma(Map<Integer, Integer> cpuInfo, double[] values) {
	                int numaNumber = 0;
	                Map<Integer, Double> cpuMap = new ConcurrentHashMap<Integer, Double>();
	                for(int index=0; index< values.length; index++) {
	                     numaNumber = cpuInfo.get(index);
			
	                     if(!cpuMap.containsKey(numaNumber)) {
				cpuMap.put(numaNumber, values[index]);
	                     }else {
				cpuMap.put(numaNumber, cpuMap.get(numaNumber) + values[index]);
	                     }
	                }
		
	                int numaCount = cpuMap.keySet().size();
	                int cpuCount =  (int)(cpuInfo.size()/numaCount);
		
	                Set<Integer> numaSet = cpuMap.keySet();
	                for(Integer numa : numaSet) {
				cpuMap.put(numa, (double)Math.round(((cpuMap.get(numa)/cpuCount)*100)*100/100.0));	 
	                }
		
	                return cpuMap;
            }

 **My database table :**
![image](https://user-images.githubusercontent.com/23719336/172276350-a985c740-6dfc-490f-be66-71f62d7cfa68.png)

**My result :** 
![image](https://user-images.githubusercontent.com/23719336/172276903-2454aef2-22e2-4560-bec3-6f356cd10765.png)

// db data -> chart (using highchart)
![image](https://user-images.githubusercontent.com/23719336/172276932-639b33b8-4181-4c4a-895b-117e4ce0045e.png)

And, Please check if the current method of obtaining cpu usage by numa is correct.

**My method :**

![image](https://user-images.githubusercontent.com/23719336/172278233-29437e82-a0de-4ab7-a7fb-8906c2d0be1b.png)

**Comment by dbwiddis** [comment]
TLDR: Windows Task Manager measures "work completed" rather than "CPU usage".  

See the [FAQ](https://github.com/oshi/oshi/blob/master/src/site/markdown/FAQ.md#why-does-oshis-system-and-processor-cpu-usage-differ-from-the-windows-task-manager).
> If you desire OSHI's output to match the Task Manager, you may optionally enable this setting in the configuration file, or by calling GlobalConfig.set(GlobalConfig.OSHI_OS_WINDOWS_CPU_UTILITY, true); shortly after startup (at least before the first instantiation of the Central Processor class). Note that OSHI will not cap its CPU Usage calculation at 100%, giving you more information than the Windows Task Manager if the "work completed" metric is important to you.

Please see #1870 for a similar report to your own, that was implemented in 6.1.6 in #1886.

**Comment by kingkong3** [comment]
By applying [GlobalConfig.set(GlobalConfig.OSHI_OS_WINDOWS_CPU_UTILITY, true); ], I was able to get the result I wanted.

Thank you, and I have an additional question another one.

        long[][] oldTicks = cpu.getProcessorCpuLoadTicks();
		
        // sleepInterval : 5000
        Thread.sleep(sleepInterval);
		
        double [] values = cpu.getProcessorCpuLoadBetweenTicks(oldTicks);

What do you mean by returned values if I set [sleepInterval] to 5000 or more?
Does the value mean the average usage in the 5-second interval?

**Comment by dbwiddis** [comment]
Yes, if you put in 5000 (milliseconds) then the returned load will be an average over that last 5 seconds.

**Comment by kingkong3** [comment]
Thank you for your kind reply.

It helped a lot!
